packages/icons/src/gcp-secret-manager.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="32" height="33" viewBox="0 0 32 33" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should we remove width and height? Also, the file looks big. did you optmize it?